docker-compose.yml

```diff
@@ -313,13 +329,14 @@ services:
         hostname: "${IOT_DASHBOARD_MAIL_HOST_NAME:-iotmail}"
         relay_ip: "${IOT_DASHBOARD_MAIL_RELAY_IP:-}"
         domain: "${IOT_DASHBOARD_MAIL_DOMAIN:-example.com}"
+        smtp_auth: "${IOT_DASHBOARD_MAIL_SMTP_AUTH:-no}"
         smtp_login: "${IOT_DASHBOARD_MAIL_SMTP_LOGIN:-}"
         smtp_password: "${IOT_DASHBOARD_MAIL_SMTP_PASSWORD:-}"
       # Enable SSL/TLS postfix communication
         tls_security_level: "${IOT_DASHBOARD_MAIL_TLS_SECURITY_LEVEL:-may}" 
         tls_wrappermode: "${IOT_DASHBOARD_MAIL_TLS_WRAPPERMODE:-no}"
-    expose:
-      - "25"
+    ports:
```

Comment:
not clear to me why port 2525 should be exposed to the outside world.

Reply:
The host system's postfix will use this `127.0.0.1:2525` as a relayhost for sending mail alert (on unattended upgrade). Also, it was not exposed to the outside world; postfix was configured in a way to allow only private subnets.

Reply:
The host system's postfix will use this `127.0.0.1:2525` as a relayhost for sending mail alert (on unattended upgrade). Also, it was not exposed to the outside world; postfix was configured in a way to allow only private subnets.